easybuild/easyconfigs/l/libecpint/libecpint-1.0.7-foss-2021a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+# easyconfig file for libecpint
+name = 'libecpint'
+version = '1.0.7'
+
+homepage = 'https://github.com/robashaw/libecpint'
+description = """Libecpint is a C++ library for the efficient evaluation of integrals over ab initio effective core potentials,
+using a mixture of generated, recursive code and Gauss-Chebyshev quadrature."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/robashaw/libecpint/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e9c60fddb2614f113ab59ec620799d961db73979845e6e637c4a6fb72aee51cc']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('pugixml', '1.11.4'),
+    ('CMake', '3.20.1')
```

Comment:
@benczaja shouldn't CMake be a builddependency, rather than a dependency?

Reply:
@smoors yes it should, thank you! I committed the change